mini-e2e-helm.groovy

```diff
@@ -198,13 +198,14 @@ node('cico-workspace') {
 		}
 		stage('run e2e') {
 			timeout(time: 120, unit: 'MINUTES') {
-				test_type = '--test-rbd=true --test-cephfs=true --test-nfs=true'
```

Comment:
Setting a default is much cleaner and easier to read as setting it in an `else` clause. Can you use a different variable name and not override the contents of the original?

Reply:
Done.